ui_framework/components/form/_color_picker.scss

```diff
@@ -0,0 +1,29 @@
+.kuiColorSwatch {
+  width: 19px;
+  height: 19px;
+  border-radius: 3px;
+  border: 1px solid black;
+  display: inline-block;
+  vertical-align: middle;
```

Comment:
Instead of using `inline-block` we can use flexbox to vertically align the content. It required adding an additional element. When I added this element I ended up adjusting the names of these classes a bit to a) use BEM conventions and b) reflect the relationships among elements a little more clearly (e.g. the connection between the popover and the cover, which I renamed overlay since we're using that term already with the modal).

Here's what the SCSS ended up looking like:

```scss
.kuiColorPicker {
  cursor: pointer;
}

  .kuiColorPicker__preview {
    display: flex;
    align-items: center;
  }

  .kuiColorPicker__swatch {
    width: 19px;
    height: 19px;
    border-radius: $globalBorderRadius;
    box-shadow: inset 0 0 0 1px rgba(#000, 0.2);
  }

  .kuiColorPicker__label {
    font-size: $globalFontSize;
    line-height: $globalLineHeight;
    margin-left: 10px;
    display: inline-block;
    vertical-align: middle;
  }

.kuiColorPickerPopUp {
  position: absolute;
  z-index: 10;
}

.kuiColorPickerPopUpOverlay {
  position: fixed;
  top: 0;
  right: 0;
  bottom: 0;
  left: 0;
}
```